### PR TITLE
Exclude some mapped FOLIO locations from being translated to Symphony codes

### LINKED ISSUE #1068: current_location field in `item_display_struct` should be the FOLIO code if we can't map it to a Symphony code for parity
We're currently lopping the library prefix off (somehow?), which breaks the downstream location mapping code for locations without a symphony equivalent (e.g. `SUL-TS-CC-REPAIR` should display in Searchworks as "Out for repair" instead of "TS-CC-REPAIR")

https://searchworks-folio-dev.stanford.edu/view/12278435
Image

### PR DESCRIPTION
**Pull request by corylown**
Fixes #1068

### CODE REVIEW

File: lib/locations_map.rb [hunk 1]
```diff
@@ -24,6 +34,9 @@ def load_home_locations
               col_sep: "\t").each_with_object({}) do |(home_location, library_code, folio_code), hash|
       library_code = { 'LANE' => 'LANE-MED' }.fetch(library_code, library_code)
 
+      # Skipping location codes that we want handle as FOLIO codes in SearchWorks.
```
Comment (jcoyne) [inline lib/locations_map.rb:37]: typo here I think.

File: lib/locations_map.rb [hunk 2]
```diff
@@ -37,6 +50,9 @@ def load_home_locations
   def load_current_locations
     CSV.parse(File.read(File.join(__dir__, 'translation_maps', 'temp_locations.tsv')),
               col_sep: "\t").each_with_object({}) do |(current_location, library_code, folio_code), hash|
+      # Skipping location codes that we want handle as FOLIO codes in SearchWorks.
```
Comment (jcoyne) [inline lib/locations_map.rb:53]: here as well

Comment (cbeer) [inline lib/locations_map.rb:53]:
```suggestion
      # Skipping location codes that we want to handle as FOLIO codes in SearchWorks.
```

File: lib/locations_map.rb (hunk 1 above)
Comment (corylown) [reply lib/locations_map.rb]: yes, thank you. Fixed

[the remark this answers is not in the document]
Comment (corylown) [reply lib/locations_map.rb]: Fixed. thank you

**Pull request merged**